### PR TITLE
minor: Updated Inputs for EmptyBlockCheck

### PR DESCRIPTION
**Pull request by MUzairS15**
Based on discussion at https://github.com/checkstyle/checkstyle/pull/11095#pullrequestreview-900959252

### CODE REVIEW

**Review by romani — APPROVED** [review]
ok to merge if CI pass

**Comment by romani** [comment]
@MUzairS15 , do you plan to create all other PRs for all your existing PRs on similar update ?
or do you want me to review all of them and ask the same question ? :) 
please pin me in PR where you think codepoint update is required to your mind.

**Comment by nick-mancuso** [comment]
Build is failing from false negatives, `EmptyBlockCheck` requires update.

**Review by nick-mancuso — CHANGES_REQUESTED** [review]
Build is failing from false negatives, `EmptyBlockCheck` requires update.

**Comment by MUzairS15** [comment]
> @MUzairS15 , do you plan to create all other PRs for all your existing PRs on similar update ?
> or do you want me to review all of them and ask the same question ? :)
> please pin me in PR where you think codepoint update is required to your mind.

I will add the output of CLI on existing PRs showing need of codepoints and if you want will create separate PRs (just input and test like in this) for those check which are not affected.

**Comment by nick-mancuso** [comment]
If check requires update -> send PR with check update and new input/UT
If check does not require input -> send PR with only new input/UT

I think this PR was premature, since we found false negative at #11095

**Comment by romani** [comment]
https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=7932&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111&l=771

```
[ERROR]   EmptyBlockCheckTest.testEmptyBlockWithEmoji:215->AbstractModuleTestSupport.verifyWithInlineConfigParser:270->AbstractModuleTestSupport.verifyViolations:427 Violation lines for /home/vsts/work/1/s/src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockWithEmoji.java differ.
missing (3): 25, 33, 45
---
expected   : [15, 25, 28, 30, 31, 33, 40, 45]
but was    : [15, 28, 30, 31, 40]

```

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockWithEmoji.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    }
+    static {
+        String c = "🎄";
+    }
+    public void fooMethod()
+    {
+        String a = "12🤩";
+        String b = "";
+        if (a == "🤩12🧐🧐") { } // violation 'Empty if block'
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockWithEmoji.java:25]: weird side effect, symbols are not even inside next to curly braces, but close enough to confuse Check

**Comment by MUzairS15** [comment]
@romani closing this PR since proof that EmptyBlock check  requires update is added in https://github.com/checkstyle/checkstyle/pull/11095

**Pull request closed without merging**